src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/ViewModels/WhenCreatedApprenticeshipListItemGroupViewModel.cs

```diff
@@ -0,0 +1,369 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using SFA.DAS.EmployerCommitments.Domain.Models.ApprenticeshipCourse;
+using SFA.DAS.EmployerCommitments.Web.ViewModels;
+
+namespace SFA.DAS.EmployerCommitments.Web.UnitTests.ViewModels
+{
+    [TestFixture]
+    public class WhenCreatedApprenticeshipListItemGroupViewModel
+    {
+        #region SetUp
+
+        private const int TestTrainingProgrammeFundingCap = 100;
+
+        private IList<ApprenticeshipListItemViewModel> _singleApprenticeship;
+        private ITrainingProgramme _testTrainingProgramme;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _singleApprenticeship = new[]
+            {
+                new ApprenticeshipListItemViewModel
+                {
+                    StartDate = new DateTime(2020,2,2),
+                    Cost = 500
+                }
+            };
+
+            _testTrainingProgramme =  new Framework
+            {
+                FundingPeriods = new[]
+                {
+                    new FundingPeriod
+                    {
+                        EffectiveFrom = new DateTime(2020, 2, 1),
+                        EffectiveTo = new DateTime(2020, 3, 1),
+                        FundingCap = TestTrainingProgrammeFundingCap
+                    }
+                },
+                EffectiveFrom = new DateTime(2020, 2, 1),
+                EffectiveTo = new DateTime(2020, 3, 1)
+            };
+        }
+
+        #endregion SetUp
+
+        #region ApprenticeshipsOverFundingLimit
+
+        [Test]
+        public void AndNoTrainingProgrammeThenThereAreNoApprenticeshipsOverFundingLimit()
+        {
+            var group = new ApprenticeshipListItemGroupViewModel(_singleApprenticeship, null);
+
+            Assert.AreEqual(0, group.ApprenticeshipsOverFundingLimit);
+        }
+
+        [Test]
+        public void AndNoApprenticeshipsThenThereAreNoApprenticeshipsOverFundingLimit()
+        {
+            var trainingProgram = new Framework();
+
+            var group = new ApprenticeshipListItemGroupViewModel(new ApprenticeshipListItemViewModel[0], trainingProgram);
+
+            Assert.AreEqual(0, group.ApprenticeshipsOverFundingLimit);
+        }
+
+        [TestCase("2020-1-15",   "99", "2020-1-10", "2020-1-20", 100, 0, Description = "StarDatet in band, cost less than cap")]
```

Comment:
typo StarDatet (and line below), I think you could also remove quotes from the second param of each TestCase.